templates/templates/_extra_metadata.html

```diff
@@ -0,0 +1,20 @@
+<meta name="theme-color" content="#E95420" />
+<meta name="twitter:account_id" content="4503599627481511" />
+<meta name="twitter:site" content="@ubuntu">
+<meta property="og:type" content="website" />
+<meta property="og:url" content="https://cn.ubuntu.com{{ request.get_full_path }}" />
+<meta property="og:site_name" content="Ubuntu" />
+
+{% if title %}
+    <meta name="twitter:title" content="{{ title }} | Ubuntu">
```

Comment:
This file seem to be indented 4 spaces instead of 2.

Reply:
I flattened it